client/my-sites/plans/current-plan/index.jsx

```diff
@@ -69,6 +77,21 @@ class CurrentPlan extends Component {
 				' its professional features.' );
```

Comment:
This tagline will be overwritten for yearly Professional plans, so this condition can be removed safely.